client/lcd/lcd_test.go

```diff
@@ -1206,6 +1210,17 @@ func getValidator(t *testing.T, port string, validatorAddr sdk.ValAddress) stake
 	return validator
 }
 
+func getValidatorDelegations(t *testing.T, port string, validatorAddr sdk.ValAddress) []stake.Delegation {
+	res, body := Request(t, port, "GET", fmt.Sprintf("/stake/validators/%s/delegations", validatorAddr.String()), nil)
```

Comment:
I feel like there should be a helper function like `StakeValidatorsDelegationsPath(validatorAddr.String())`

https://guides.rubyonrails.org/routing.html#path-and-url-helpers

In Ruby, you can use [magic](https://media.tenor.co/images/7f6402cf7df54cdf24505ee7895326aa/raw) to create such functions on the fly. In Go, we'd need to write them ourselves. Still, there may be a benefit in having them.